.github/workflows/ci.yaml

```diff
@@ -56,14 +56,14 @@ jobs:
       - name: Install snap file
         run: |
           sudo snap install charmed-kafka_${{ needs.build.outputs.version }}_amd64.snap --dangerous
-          sudo snap install charmed-zookeeper --channel=3/edge
+          sudo snap install charmed-zookeeper --channel=3/edge/dpe-2196-logging
```

Comment:
**question:** Remind me, is this the 'temporary' deploy that's needed?
If not, I reckon this should be removed.

Reply:
Indeed! Thanks for flagging this! I would have most likely forgotten about this. 

I'll merge the zookeeper one and then update this, once the new snap will be published!